docs/docs/quickstart/010.install.mdx

```diff
@@ -11,42 +11,18 @@ The script will in turn download required dependencies and copy them to the bin
 <Tabs groupId="platforms">
 <TabItem value="linux_macos" label="Linux/MacOS">
 
-``` shell
-
-$ mkdir $HOME/starlake
-$ cd $HOME/starlake
-$ bash  < (curl -s https://raw.githubusercontent.com/starlake-ai/starlake/master/setup.sh)
-
-    _____ _______       _____  _               _  ________
-    / ____|__   __|/\   |  __ \| |        /\   | |/ /  ____|
-    | (___    | |  /  \  | |__) | |       /  \  | ' /| |__
-    \___ \   | | / /\ \ |  _  /| |      / /\ \ |  < |  __|
-    ____) |  | |/ ____ \| | \ \| |____ / ____ \| . \| |____
-    |_____/   |_/_/    \_\_|  \_\______/_/    \_\_|\_\______|
-
-
-    Where do you want to install Starlake? [/Users/me/starlake]: /Users/me/starlake
-    Last 5 available versions:
-    1.0.1-SNAPSHOT
-    1.0.0
-    0.8.1
-    0.8.0
-    0.7.4
-    0.7.3
-    Which version do you want to install? [1.0.0]:
-
-
+```shell
+sh <(curl https://raw.githubusercontent.com/starlake-ai/starlake/master/distrib/setup.sh)
+starlake help
```

Comment:
@hayssams i removed the example, should we leave it ?

Reply:
@elarib yes!
Better leave it

Reply:
I mean keep the changes you made